src/Storefront/Page/Address/Detail/AddressDetailPageLoader.php

```diff
@@ -104,14 +106,14 @@ private function getSalutations(SalesChannelContext $salesChannelContext): Salut
     private function getCountries(SalesChannelContext $salesChannelContext): CountryCollection
     {
         $criteria = (new Criteria())
-            ->addFilter(new EqualsFilter('country.active', true))
```

Comment:
same here

Reply:
Yes, that's already default of the `CountryRoute`